PollBuddy-Server/backend/routes/users.js

```diff
@@ -371,11 +374,11 @@ router.post("/register", function (req, res) {
           }
 
         } else {
-
+          
           // One result changed, therefore it worked.
           // Configure user data and save in session
           req.session.userData = {};
-          req.session.userData.userID = result.insertedId.str;
+          req.session.userData.userID = result.insertedId;
```

Comment:
As long as this works, let's keep it and other usages then don't have to convert to object ID from string, like they currently do.